src/components/TruckSizeSelect/TruckSizeSelect.css

```diff
@@ -22,5 +24,6 @@
     flex-direction: row;

```

Comment:
Oh can we now drop the whole `truck-size-container` ? That would be awesome. IMHO it would be great to get to place where component is only one file. No extra CSS, just composed of general CSS classes from bootstrap + in App.scss if we need something extra.

Reply:
Yass, that does sound simpler in a way, and better, and also guarantees more uniformity ^^ I am counting on doing a restyling later anyway, let's do that too then.